src/screens/GroupInfo/index.js

```diff
@@ -50,7 +50,7 @@ const GroupInfo = () => {
     selectedUser,
     handleCloseSelectUser,
     openModal,
-    alertRemoveUser,
+    handleOpenPopup,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `alertRemoveUser` has been renamed to `handleOpenPopup`. This is a good change as it improves the readability of the code by making the function name more descriptive of its action. However, ensure that this renaming is reflected everywhere in the codebase where this function is used to maintain consistency and avoid potential errors.
```diff
-    alertRemoveUser,
+    handleOpenPopup,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->